JT/Example of README.md

```diff
@@ -0,0 +1,411 @@
+# Front-End Unit Test Workshop - Fixers
+
+### A. Error: Jest encountered an unexpected token
+Configure jest to compile ts files.
+
+<details>
+<summary>Solution</summary>
+
+- install **babel/preset-env**:
+- install **babel/preset-react**
+- install **babel/preset-typescript**
+```shell
+npm install -D @babel/preset-env
+npm install -D @babel/preset-react
+npm install -D @babel/preset-typescript
+```
+- add file **babel-jest.config.js** to project
+```js
+module.exports = {
+  presets: ['@babel/preset-env', '@babel/preset-react']
+}
+```
+- add file **babel.config.js** to project
+```js
+module.exports = {
+  presets: [['@babel/preset-env', { targets: { node: 'current' } }],
+    ["@babel/preset-react", {"runtime": "automatic"}]],
+}
+```
+</details>
+
+### B'. Error: Test environment jest-environment-jsdom cannot be found
+Install missing library.
+
+<details>
+<summary>Solution</summary>
+
+- install **jest-environment-jsdom**:
+```shell
+npm install -D jest-environment-jsdom
+```
+</details>
+
+### B". Error: Document is not defined
```

Comment:
Duplicating section B.

Reply:
ther's two B in  README example, with `'` sign to define first and second step in this way - B' and B''